src/modules/api/GameClass.js

```diff
@@ -0,0 +1,61 @@
+import { API } from './api.js';
+import { encodeHTMLEntities } from './util.js';
+
+export default class Game extends API {
+  constructor() {
+    super();
+    this.addSuccess = 'Leaderboard score created correctly.';
+    this.data = [];
+  }
+
+  add = () => {
+    const form = document.getElementById('add-score-form');
+    form.addEventListener('submit', async (e) => {
+      e.preventDefault();
+      await this.addApi({
+        user: e.target.children[0].value,
+        score: e.target.children[1].value,
+      })
+        .then((res) => {
+          if (res === this.addSuccess) {
+            e.target.reset();
+            this.display();
+            this.msg('Successfully Added', true);
+          }
+        }).catch((error) => {
+          this.msg('Successfully Failed', false);
+          throw new Error(error);
+        });
+    });
+  }
+
+  refresh = async () => {
+    this.data = await this.refreshApi();
+    return this.data;
+  }
+
+  display = async () => {
+    const scoreTable = document.querySelector('.score-table');
+    scoreTable.innerHTML = '';
+    await this.refresh().then((res) => {
+      res.forEach((score) => {
+        scoreTable.innerHTML += this.score(score);
+      });
+    });
+  }
```

Comment:
- [ ] While trying to test your application in order to confirm whether the app is working as expected, I noticed that I could neither hit the api to create user and its score nor hit the api to retrieve the data from the get api endpoint. Kindly look into this and fix it. Below is the screen record of how your app behave at this moment.

https://user-images.githubusercontent.com/26597432/201712084-55755494-573f-45b8-a6ba-d5150f0c458e.mp4